docs/madr/decisions/003-logging.md

```diff
@@ -0,0 +1,245 @@
+# Logging standard
+
+* Status: accepted
+
+## Context and Problem Statement
+
+Seamless troubleshooting is a feature. We should keep more attention to the logging in Kuma.
+
+## Considered Options
+
+* Adopting consistent logging standard.
+
+## Decision Outcome
+
+Chosen option: "Adopting consistent logging standard", because it's the only option.
+
+### Positive Consequences
+
+* Better UX for users.
+* Better troubleshooting means both users and maintainers spent less time debugging an issue.
+
+### Negative Consequences
+
+* More effort to try to follow the guidelines.
+
+## Pros and Cons of the Options
+
+### Adopting consistent logging standard
+
+#### Convention
+
+Ideally, all those points are linted.
```

Comment:
Seems like it's technically feasible: https://golangci-lint.run/contributing/new-linters/

Reply:
https://github.com/kumahq/kuma/issues/4513 created issue